model-server/src/main/kotlin/org/modelix/model/server/handlers/ContentExplorer.kt

```diff
@@ -100,7 +129,21 @@ class ContentExplorer(private val client: IModelClient, private val repoManager:
         }
     }
 
-    private fun FlowContent.contentPageBody(rootNode: PNodeAdapter, versionHash: String) {
+    private fun collectExpandableChildNodes(node: PNodeAdapter, expandedNodeIds: Set<String>): Set<String> {
```

Comment:
The name is much better now. The idea of the arguments is that we look for nodes to expand under `node` and `expandedNodeIdes` is the state of nodes that are already expanded from previous interactions? If so, maybe the arguments can be renamed to better reflect this:

```suggestion
    private fun collectExpandableChildNodes(from: PNodeAdapter, alreadyExpandedNodeIds: Set<String>): Set<String> {
```

Reply:
or maybe `parent` or `under`?

Reply:
Parameters are renamed.